src/main/proto/ProgramService.proto

```diff
@@ -156,7 +157,7 @@ message ListUserRequest {
 }
 
 message ListUserResponse {
-  repeated Invitation invitations = 1;
+  repeated UserStatus invitations = 1;
```

Comment:
why not `repeated UserStatus users = 1;` ?

Reply:
Because I missed that. :-( Thanks for catching it!